### ISSUE TITLE
Error during installation

### ISSUE THREAD

**Issue by ravindk89** [maintainer-org/maintainer#42]
Getting the following error trying to install via go 1.16.5 on Manjaro 20.

```shell
go install github.com/maintainer-org/maintainer@latest
go: downloading github.com/maintainer-org/maintainer v1.0.5
go: finding module for package github.com/gaocegege/maintainer/cmd
go: downloading github.com/gaocegege/maintainer v1.0.5
go: found github.com/gaocegege/maintainer/cmd in github.com/gaocegege/maintainer v1.0.5
go: finding module for package golang.org/x/oauth2
go: finding module for package github.com/spf13/cobra
go: finding module for package github.com/google/go-github/github
go: finding module for package github.com/spf13/viper
go: downloading github.com/spf13/cobra v1.2.1
go: downloading github.com/spf13/viper v1.8.1
go: downloading github.com/google/go-github v17.0.0+incompatible
go: downloading golang.org/x/oauth2 v0.0.0-20210628180205-a41e5a781914
go: found github.com/google/go-github/github in github.com/google/go-github v17.0.0+incompatible
go: found github.com/spf13/cobra in github.com/spf13/cobra v1.2.1
go: found github.com/spf13/viper in github.com/spf13/viper v1.8.1
go: found golang.org/x/oauth2 in golang.org/x/oauth2 v0.0.0-20210628180205-a41e5a781914
go: downloading github.com/magiconair/properties v1.8.5
go: downloading github.com/pelletier/go-toml v1.9.3
go: downloading github.com/spf13/afero v1.6.0
go: downloading golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4
go: downloading golang.org/x/text v0.3.5
go: finding module for package github.com/google/go-querystring/query
go: downloading github.com/google/go-querystring v1.1.0
go: found github.com/google/go-querystring/query in github.com/google/go-querystring v1.1.0
# github.com/gaocegege/maintainer/contributing
mygo/pkg/mod/github.com/gaocegege/maintainer@v1.0.5/contributing/contributing.go:109:47: not enough arguments in call to client.Repositories.Get
        have (string, string)
        want (context.Context, string, string)
```

I asked some co-workers to check and they ran into the same issues using both this and `GO111MODULE=on go get github.com/maintainer-org/maintainer`. 

There was a migration on April 20th, maybe something broke around then? I also can't get the Docker image to work (yells about config files).

Guidance appreciated, thanks :)

**Comment by gaocegege** [comment]
Thanks for your report, I will have a look ASAP.

**Comment by gaocegege** [comment]
@ravindk89 The problem was fixed in #43

**Comment by ravindk89** [comment]
Much appreciated!
